### DISCUSSION TITLE
problem following theme update

### DISCUSSION THREAD

**Discussion by HDInfautre** [templaza/astroid-framework#923] [Q&A] [answered]
Hello
I have a big problem after updating astroid on a test site
https://test.altius.fr/fr/accompagner.html
To compare with the site without the updates:
https://www.altius.fr/fr/accompagner.html
...

Can you look to help me solve this?
I give direct access if necessary:

Regards

**Comment by sonvnn** [comment, marked as answer]
The style was changed. You can change your css 
from
```
.full-image, #ca-1529739248826 > main > div > figure img, .img-fluid {
    display: none !important;
}
```
To 
```
.astroid-article-full-image {
display: none !important;
}
```
